### PR TITLE
Remove information overload from toctrees

### PR DESCRIPTION
**Pull request by yukiisbored**
With the change:
image

What it looks like currently:
image

### CODE REVIEW

File: source/tutorials/index.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Tutorials
 
+This section contains a collection of articles on how to achieve a goal with the Nix ecosystem.
```
Comment (fricklerhandwerk) [inline source/tutorials/index.md:3]:
```suggestion
This section contains a collection of articles on how to get thing done in the Nix ecosystem.
```

**Review by fricklerhandwerk — COMMENTED** [review]
I think removing it altogether is too much. If you view it on mobile, having the collapsed menu as the only way to navigate is not optimal. There should be at least a top-level navigation on the landing page. Otherwise I'm in favor of the reduction of noise in general.

@domenkozar @lucperkins please also take a look.

File: source/tutorials/index.md (hunk 1 above)
Comment (lucperkins) [reply source/tutorials/index.md:3]: s/thing/things

Comment (fricklerhandwerk) [reply source/tutorials/index.md:3]: Feel free to ninja-edit my typo-ridden suggestions.

**Comment by yukiisbored** [comment]
@fricklerhandwerk I changed it so it shows the top-level categories on the frontpage.

image

**Review by fricklerhandwerk — APPROVED** [review]
Nice!

**Pull request merged**